package.json

```diff
@@ -50,5 +52,10 @@
       "@babel/preset-env",
       "@babel/preset-typescript"
     ]
+  },
+  "prepush-if-changed": {
+    "example/project-nextjs-tailwind/**/*": "cd example/project-nextjs-tailwind && npm test && cd ../..",
+    "example/project-nextjs-mui/**/*": "cd example/project-nextjs-mui && npm test && cd ../..",
+    "example/project-elixir-tailwind/**/*": "cd example/project-elixir-tailwind && mix test && cd ../.."
```

Comment:
Looks like a solid change. I wonder if we can make this programmatic so we don't need to update this for every project.

Great fix for now though!